packages/styles/scss/components/masthead/_masthead-search.scss

```diff
@@ -0,0 +1,132 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin masthead-search {
```

Comment:
Did we just move this out into its own file? Did anything change in here?

Reply:
Yep just moving it out to its own file.